### PR TITLE
🦺[Tech Debt] Fixed podfile: Firebase cocoapod

### PR DESCRIPTION
**Pull request by kcw-grunt**
- Cleaned up podfile
- Turning off the code coverage allows the SwiftUI canvas to work

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -8,9 +8,8 @@ platform :ios, '13.0'
 def shared_pods
   pod 'UnstoppableDomainsResolution', '~> 0.3.6'
   pod 'KeychainAccess', '~> 4.2'
-  pod 'Firebase/Analytics', '~> 6.0'
-  pod 'Firebase/Crashlytics', '~> 6.0'
-  # add pod 'SwiftLint'
+  pod 'Firebase/Analytics'
```
Comment (kcw-grunt) [inline Podfile:11]: Bad edit from months ago.  This updates the Firebase pod

File: loafwallet.xcodeproj/xcshareddata/xcschemes/loafwallet.xcscheme [hunk 2]
```diff
@@ -28,7 +28,6 @@
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       shouldUseLaunchSchemeArgsEnv = "YES"
       enableASanStackUseAfterReturn = "YES"
-      codeCoverageEnabled = "YES"
```
Comment (kcw-grunt) [inline loafwallet.xcodeproj/xcshareddata/xcschemes/loafwallet.xcscheme]: This is a Xcode bug.  You can't run code coverage **and** run SwiftUI canvas

**Comment by JCHeist** [comment]
Looks good other than the test faking for some reason. Why is that?

**Comment by kcw-grunt** [comment]
> Looks good other than the test faking for some reason. Why is that?

Bitrise has a step that can run pod install or use the cache., based on the Podfile.lock.  I need to take a look..

**Review by JCHeist — APPROVED** [review]
Reran test on Bitrise, Tests now passing, if this PR is still applicable, good to merge now.

**Pull request merged**